### PR TITLE
Add LLM selection drop down for Cody Enterprise

### PR DESCRIPTION
**Pull request by MaedahBatool**
- https://linear.app/sourcegraph/issue/CODY-2870/update-enterprise-llm-selection-dropdown-docs

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@kalanchan are we good to merge this one? Is this the right time now?

File: docs/cody/clients/install-jetbrains.mdx [hunk 1]
```diff
@@ -151,7 +151,7 @@ Like the Edit Code and Generate Unit Test commands, you can generate inline docu
 
 ## Supported LLM models
 
-Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Both Cody Free and Pro users can choose from a list of supported LLM models for Chat and Commands.
+Users on Cody **Free**, **Pro**, and **Enterprise** can choose from a list of supported LLM models for Chat and Commands. Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands.
```
Comment (jamesmcnamara) [inline docs/cody/clients/install-jetbrains.mdx:154]: Isn't it Claude Sonnet 3.5?

**Review by jamesmcnamara — APPROVED** [review]
One small nit and then it looks good!

File: docs/cody/clients/install-jetbrains.mdx (hunk 1 above)
Comment (jamesmcnamara) [reply docs/cody/clients/install-jetbrains.mdx:154]: image

**Comment by aramaraju** [comment]
As mentioned on the DM with @MaedahBatool, let's hold off on these docs until I have more clarity here on what state this will come out on Aug 7th and the limitations

**Comment by aramaraju** [comment]
@MaedahBatool Let's kill this branch, eh?

**Pull request closed without merging**